src/lC_common_h.jl

```diff
@@ -1,705 +1,1256 @@
+const CURLE_FTP_COULDNT_SET_BINARY = CURLE_FTP_COULDNT_SET_TYPE
+const CURLE_FTP_QUOTE_ERROR = CURLE_QUOTE_ERROR
+const CURLE_TFTP_DISKFULL = CURLE_REMOTE_DISK_FULL
+const CURLE_TFTP_EXISTS = CURLE_REMOTE_FILE_EXISTS
+const CURLE_HTTP_RANGE_ERROR = CURLE_RANGE_ERROR
+const CURLE_FTP_SSL_FAILED = CURLE_USE_SSL_FAILED
+const CURLE_OPERATION_TIMEOUTED = CURLE_OPERATION_TIMEDOUT
+const CURLE_HTTP_NOT_FOUND = CURLE_HTTP_RETURNED_ERROR
+const CURLE_HTTP_PORT_FAILED = CURLE_INTERFACE_FAILED
+const CURLE_FTP_COULDNT_STOR_FILE = CURLE_UPLOAD_FAILED
+const CURLE_FTP_PARTIAL_FILE = CURLE_PARTIAL_FILE
+const CURLE_FTP_BAD_DOWNLOAD_RESUME = CURLE_BAD_DOWNLOAD_RESUME
+const CURLE_ALREADY_COMPLETE = 99999
+const CURLOPT_FILE = CURLOPT_WRITEDATA
+const CURLOPT_INFILE = CURLOPT_READDATA
+const CURLOPT_WRITEHEADER = CURLOPT_HEADERDATA
+const CURLOPT_WRITEINFO = CURLOPT_OBSOLETE40
+const CURLOPT_CLOSEPOLICY = CURLOPT_OBSOLETE72
+
+# Skipping MacroDefinition: CURLAUTH_NONE ( ( unsigned long ) 0 )
+# Skipping MacroDefinition: CURLAUTH_BASIC ( ( ( unsigned long ) 1 ) << 0 )
+# Skipping MacroDefinition: CURLAUTH_DIGEST ( ( ( unsigned long ) 1 ) << 1 )
+# Skipping MacroDefinition: CURLAUTH_NEGOTIATE ( ( ( unsigned long ) 1 ) << 2 )
+
+const CURLAUTH_GSSNEGOTIATE = CURLAUTH_NEGOTIATE
+const CURLAUTH_GSSAPI = CURLAUTH_NEGOTIATE
+
+# Skipping MacroDefinition: CURLAUTH_NTLM ( ( ( unsigned long ) 1 ) << 3 )
+# Skipping MacroDefinition: CURLAUTH_DIGEST_IE ( ( ( unsigned long ) 1 ) << 4 )
+# Skipping MacroDefinition: CURLAUTH_NTLM_WB ( ( ( unsigned long ) 1 ) << 5 )
+# Skipping MacroDefinition: CURLAUTH_BEARER ( ( ( unsigned long ) 1 ) << 6 )
+# Skipping MacroDefinition: CURLAUTH_ONLY ( ( ( unsigned long ) 1 ) << 31 )
+
+const CURLAUTH_ANY = ~CURLAUTH_DIGEST_IE
+
+# Skipping MacroDefinition: CURLAUTH_ANYSAFE ( ~ ( CURLAUTH_BASIC | CURLAUTH_DIGEST_IE ) )
+
+const CURLSSH_AUTH_ANY = ~0
+const CURLSSH_AUTH_NONE = 0
+const CURLSSH_AUTH_PUBLICKEY = 1 << 0
+const CURLSSH_AUTH_PASSWORD = 1 << 1
+const CURLSSH_AUTH_HOST = 1 << 2
+const CURLSSH_AUTH_KEYBOARD = 1 << 3
+const CURLSSH_AUTH_AGENT = 1 << 4
+const CURLSSH_AUTH_GSSAPI = 1 << 5
+const CURLSSH_AUTH_DEFAULT = CURLSSH_AUTH_ANY
+const CURLGSSAPI_DELEGATION_NONE = 0
+const CURLGSSAPI_DELEGATION_POLICY_FLAG = 1 << 0
+const CURLGSSAPI_DELEGATION_FLAG = 1 << 1
+const CURL_ERROR_SIZE = 256
+const CURLSSLOPT_ALLOW_BEAST = 1 << 0
+const CURLSSLOPT_NO_REVOKE = 1 << 1
+const CURL_HET_DEFAULT = Int32(200)
+
+# begin enum ANONYMOUS_9
+const ANONYMOUS_9 = UInt32
+const CURLUSESSL_NONE = (UInt32)(0)
+const CURLUSESSL_TRY = (UInt32)(1)
+const CURLUSESSL_CONTROL = (UInt32)(2)
+const CURLUSESSL_ALL = (UInt32)(3)
+const CURLUSESSL_LAST = (UInt32)(4)
+# end enum ANONYMOUS_9
+
+const CURLFTPSSL_NONE = CURLUSESSL_NONE
+const CURLFTPSSL_TRY = CURLUSESSL_TRY
+const CURLFTPSSL_CONTROL = CURLUSESSL_CONTROL
+const CURLFTPSSL_ALL = CURLUSESSL_ALL
+const CURLFTPSSL_LAST = CURLUSESSL_LAST
+const curl_usessl = Void
```

Comment:
`Void` no longer exists on Julia 1.0. This looks like Clang.jl needs to switch to using `Cvoid`

Reply:
Looks like that was done here: `https://github.com/ihnorton/Clang.jl/commit/1ec7d258f60725fd25c6691e2806e6b5528ce63b` I will try again using master (was on a release).